dashboard/public/pixelation/pixelation.js

```diff
@@ -100,7 +100,7 @@ function initProjects() {
 
 function pixelationDisplay() {
   pixel_data.value = options.projectData || options.data;
-  drawGraph();
+  drawGraph(null, false, true /* updateControls */);
```

Comment:
I thought inline parameter comments were in the style guide, but they are not. I've removed them.